home/urls.py

```diff
@@ -7,4 +7,5 @@
 urlpatterns = [
     path("", views.home_view, name="home"),
     path("search", views.search_view, name="search"),
+    path("details", views.details_view, name="details"),
```

Comment:
We should refactor the details url pattern to accept an id/pk value like this. 
`path(details/<str:id>/` . 
We can then build the individual url in the search results likes this. `"{% url 'home:details' id=result.id %}"`